pkg/cmd/server/api/helpers.go

```diff
@@ -120,6 +121,10 @@ func GetMasterFileReferences(config *MasterConfig) []*string {
 	refs = append(refs, &config.ServingInfo.ServerCert.CertFile)
 	refs = append(refs, &config.ServingInfo.ServerCert.KeyFile)
 	refs = append(refs, &config.ServingInfo.ClientCA)
+	for i := range config.ServingInfo.NamedCertificates {
```

Comment:
I only added references for the ones we support... add to HTTPServingInfo instead? felt weird there